TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -106,7 +68,10 @@ function Execute-SqlQueryDeployment
             }
         }
 
-        Invoke-Sqlcmd @spaltArguments
+        $additionalArguments = $additionalArguments.Replace("`$", "```$")
+
+        Write-Verbose "Invoke-SqlCmd arguments : $spaltArgumentsToLogJson  $additionalArguments"
+        Invoke-Expression "Invoke-SqlCmd @spaltArguments $additionalArguments"
```

Comment:
What If additional arguments has -option "Val ue", here I am seeing to invoke expression we are sending -option Val ue which may not be the one we want.